packages/common-ui-web/src/components/InspectionGallery/hooks/useInspectionGalleryItems.ts

```diff
@@ -53,6 +53,7 @@ function compareGalleryItems(
 function getItems(
   inspectionId: string,
   captureMode: boolean,
+  enableAddDamage: boolean | undefined,
```

Comment:
Put the parameter last and replace `enableAddDamage: boolean | undefined` with `enableAddDamage?: boolean`